### PR TITLE
Testcase verifies schedule backup creation with all namespaces

### PR DESCRIPTION
**Pull request by vpinisetti-px**
**What this PR does / why we need it**:
This testcase verifies schedule backup creation with all namespaces

**Which issue(s) this PR fixes** (optional)
Closes # https://portworx.atlassian.net/browse/PA-502

**Special notes for your reviewer**:
http://aetos.pwx.purestorage.com/resultSet/testSetID/91385

### CODE REVIEW

**Review by snigdha-px — APPROVED** [review]
LGTM

File: tests/backup/backup_test.go [hunk 1]
```diff
@@ -7075,6 +7075,132 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
...
 })
 
+// This testcase verifies schedule backup creation with all namespaces.
+var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
+	var contexts []*scheduler.Context
+	var appContexts []*scheduler.Context
+	var backupLocationName string
+	var backupLocationUID string
+	var cloudCredUID string
+	var bkpNamespaces []string
```
Comment (lsrinivas-pure) [inline tests/backup/backup_test.go:7085]: can you declare one var and name all the variables in it
for ex:

var (
contexts []*scheduler.Context
appContexts []*scheduler.Context
backupLocationName string
)

File: tests/backup/backup_test.go [hunk 2]
```diff
@@ -7075,6 +7075,132 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
...
+			for _, provider := range providers {
+				cloudAccountName = fmt.Sprintf("%s-%v", provider, timeStamp)
+				cloudCredUID = uuid.New()
+				cloudCredUIDMap[cloudCredUID] = cloudAccountName
+				CreateCloudCredential(provider, cloudAccountName, cloudCredUID, orgID)
+			}
+		})
+
+		Step("Adding Backup Location", func() {
+			providers := getProviders()
```
Comment (lsrinivas-pure) [inline tests/backup/backup_test.go:7134]: if there is no change in providers, declare it at testcase level and reuse ir

File: tests/backup/backup_test.go [hunk 3]
```diff
@@ -7075,6 +7075,132 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
...
+		scheduleUid, err := GetScheduleUID(backupName, orgID, ctx)
+		log.FailOnError(err, "Error while getting schedule uid %v", backupName)
+		err = DeleteSchedule(backupName, scheduleUid, periodicPolicyName, schPolicyUid, orgID)
+		log.FailOnError(err, "Error deleting backup schedules %v", backupName)
+		log.Info("Deleting restores")
+		err = DeleteRestore(restoreName, orgID, ctx)
+		dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying restore %s deletion", restoreName))
+		log.Info("Deleting the deployed apps after test execution")
+		for i := 0; i < len(contexts); i++ {
+			opts := make(map[string]bool)
```
Comment (lsrinivas-pure) [inline tests/backup/backup_test.go:7193]: move the opts out of for loop

File: tests/backup/backup_test.go (hunk 1 above)
Comment (vpinisetti-px) [reply tests/backup/backup_test.go]: Done

File: tests/backup/backup_test.go (hunk 3 above)
Comment (vpinisetti-px) [reply tests/backup/backup_test.go]: Done

File: tests/backup/backup_test.go (hunk 2 above)
Comment (vpinisetti-px) [reply tests/backup/backup_test.go]: Done

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Review by mkoppal-px — APPROVED** [review]
LGTM 👍

**Pull request merged**